core/object/object.h

```diff
@@ -525,8 +525,8 @@ class Object {
 	Array _get_signal_list() const;
 	Array _get_signal_connection_list(const String &p_signal) const;
 	Array _get_incoming_connections() const;
-	void _set_bind(const String &p_set, const Variant &p_value);
-	Variant _get_bind(const String &p_name) const;
+	void _set_bind(const StringName &p_set, const Variant &p_value);
```

Comment:
I wonder if `_add_user_signal` and `_get_signal_connection_list` above should also use StringName?

Reply:
It makes sense for `_get_signal_connection_list()`. I will update the PR.